### ISSUE TITLE
Initial comparison of distribution vs abundance trends in moths

### ISSUE THREAD

**Issue by drnickisaac** [BiologicalRecordsCentre/BRCtracker#3]
Dependent on #1 
@colinharrower what is the latest news on abundance trends from RIS?

**Comment by drnickisaac** [comment]
RIS moth initial analysis is here:
P:\NEC05829 LTS-M-ASSIST\WP2_Bell\Data_WP2\Rothamsted Insect Survey Analysis

This should give me enough to get started. @colinharrower to draw up a spreadsheet indicating which species have problems.

**Comment by colinharrower** [comment]
I found the comparisons I also did comparing the GAI results with those from the State of Moths report replicating what I did with the Conrad vs GAI comparisons. I've put the relevant files (1 spreadsheet and 2 pdfs) in the p drive folder with the data from the first runs and the existing Conrad comparisons.

I also noticed that I had already created text files containing all of the collated index values for all species/year combinations (`final_ci_vals.txt`) along with a file that contains all of the site index values for each species/year/site combination (`final_si_vals.txt`). These files are also in the folder containing the results from the first GAI run

**Comment by drnickisaac** [comment]
Colin has now completed a second cut of GAI models (#13): 297 species are useable without further investigation. Next step is to compare the two.
Abundance trends: P:\NEC05829 LTS-M-ASSIST\WP2_Bell\WP2.3.5_Moth trends\Summary Results_Figures
Distribution trends: W:\PYWELL_SHARED\Pywell Projects\BRC\Tom August\ASSIST\RIS_Moths   

- [ ] Setup RStudio project
- [ ] Write a script to extract date from both sets of outputs
- [ ] Plots and summary statistics

**Comment by drnickisaac** [comment]
I am working here: P:\NEC04273_SpeciesDistribution\Workfiles\BRC Range Trends\Abundance-RangeTrend
(There has been an RStudio project here since May 2014!)

**Comment by drnickisaac** [comment]
This issue is now superseded by a new one with both Distribution-Abundance comparison and Structured-Unstructured.

**Comment by JackHHatfield91** [comment]
close #2
